src/ui/components/NetworkMonitor/RequestTable.tsx

```diff
@@ -26,6 +27,7 @@ const RequestTable = ({
   const { columns, getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } = table;
 
   const onSeek = (request: RequestSummary) => {
+    trackEvent("netMonitor.seek_to_request");
```

Comment:
maybe standardize on `netMonitor` vs. `netmonitor`. Do you think this would be worth pulling out into constants?  (I think maybe I saw a PR from @abstractalgo  about this?)

Reply:
I did mention in the Discord thread ([private thread](https://discord.com/channels/779097926135054346/934430769135362049)) that we most definitely should add type safety around tracking.